File: lab4 mpp/TeledonDomain/Voluntar.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace teledon.domain
{
    public class Voluntar : Entity<long>
    {
        public string Username { get; set; }
        public string Password { get; set; }

        public Voluntar(string Username, string Password)
        {
            this.Username = Username;
            this.Password = Password;
        }

    }
}

File: GUImpp/TeledonModel/src/main/java/teledon/model/Voluntar.java
package teledon.model;

public class Voluntar extends Entity<Long> {

    private String username;
    private String password;


    public Voluntar(String username, String password) {
        this.username = username;
        this.password = password;
    }

    @Override
    public String toString() {
        return "Voluntar{" +
                "username='" + username + '\'' +
                ", password='" + password + '\'' +
                '}';
    }



    public Voluntar() {

    }


    public String getUsername() {
        return username;
    }

    public void setUsername(String username) {
        this.username = username;
    }

    public String getPassword() {
        return password;
    }

    public void setPassword(String password) {
        this.password = password;
    }


}
